daemon/state.go

```diff
@@ -281,7 +281,7 @@ func (d *Daemon) regenerateRestoredEndpoints(state *endpointRestoreState) (resto
 				return
 			}
 
-			ep.LogStatusOKLocked(endpoint.Other, "Synchronizing endpoint labels with KVStore")
+			ep.SetStateLocked(endpoint.StateRestoring, "Synchronizing endpoint labels with KVStore")
```

Comment:
I thought about setting this as `endpoint.StateWaitingForIdentity()`, but that would require thinking through a few more state machine transitions so this seemed like a safer alternative.